packages/wpcom-proxy-request/src/index.js

```diff
@@ -220,6 +220,22 @@ export function requestAllBlogsAccess() {
  */
 
 function submitRequest( params ) {
+	// Sometimes the `iframe.contentWindow` is `null` even though the `iframe` has been correctly
+	// loaded. Can happen when some other buggy script removes it from the document.
+	if ( ! iframe.contentWindow ) {
+		debug( 'proxy iframe is not present in the document' );
+		// Look up the issuing XHR request and make it fail
+		const id = params.callback;
+		const xhr = requests[ id ];
+		delete requests[ id ];
+		reject(
+			xhr,
+			new WPError( { status_code: 500, error_description: 'proxy iframe element is not loaded' } ),
```

Comment:
Do we really need the `new` here? Seems like the error object is actually instantiated inside and returned by the function, so there's no need to make the `WPError` an actual instance too.

```suggestion
			WPError( { status_code: 500, error_description: 'proxy iframe element is not loaded' } ),
```

Reply:
Good catch, `WPError` is indeed not a class constructor, it's a factory function that creates `Error`s. Fixed.